tests/restart/retrying/suite.rc

```diff
@@ -1,117 +1,34 @@
 #!jinja2
-{%- if BATCH_SYS_NAME is defined and BATCH_SYS_NAME %}
-{%- set HOST = environ['CYLC_TEST_BATCH_TASK_HOST'] %}
-{%- set SITE_DIRECTIVES = environ['CYLC_TEST_BATCH_SITE_DIRECTIVES'] %}
-{%- else %}
-{%- set BATCH_SYS_NAME = none %}
-{%- set HOST = "" %}
-{%- set SITE_DIRECTIVES = "" %}
-{%- endif %}
 {%- set TEST_DIR = environ['TEST_DIR'] %}
 [cylc]
+    UTC mode = True
     [[event hooks]]
-        timeout handler = "touch {{ TEST_DIR }}/suite-stopping && shutdown_this_suite_hook"
-        timeout = 3
+        timeout handler = shutdown_this_suite_hook
+        timeout = PT3M
 [scheduling]
-    initial cycle time = 2013092300
-    final cycle time   = 2013092306
-    runahead limit = 2
+    initial cycle time = 20130923T00
+    final cycle time   = 20130923T00
     [[dependencies]]
-        [[[0,6,12,18]]]
+        [[[R1]]]
             graph = """
-                tidy[T-6] => retrying_task
-                retrying_task:start => force_restart
-                force_restart => output_states
-                output_states & retrying_task => tidy
+                retrying_task:start => shutdown
+                shutdown => output_states
+                output_states & retrying_task => finish
             """
 [runtime]
-    [[root]]
-        script = "sleep 1"
-    [[RESTART_TASKS]]
-        {%- if BATCH_SYS_NAME %}
-        [[[job submission]]]
-            method = {{ BATCH_SYS_NAME }}
-        [[[directives]]]
-        {%- if BATCH_SYS_NAME == 'loadleveler' %}
-            class            = serial
-            job_type         = serial
-            wall_clock_limit = '20,15'
-        {%- elif BATCH_SYS_NAME == 'slurm' %}
-            --time = 20
-        {%- elif BATCH_SYS_NAME == 'pbs' %}
-            -l walltime=120
-        {%- endif %}
-            {{ SITE_DIRECTIVES }}
-        [[[remote]]]
-            host = {{ HOST }}
-        {%- endif %}
     [[retrying_task]]
-        inherit = RESTART_TASKS
         script = """
+            sleep 1
```

Comment:
`03-retrying.t` fails in my environment because this task goes to the `retrying` state before `shutdown` can trigger off its `running` state.  Changing this line to `sleep 10` fixes it.